.github/workflows/build_cc_target.yml

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: Build check
+
+on: [push, pull_request]
+
+jobs:
+  targets:
+    name: Build GCC test target
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        gcc: ['12.2.Rel1', '11.3.Rel1', '10.3-2021.10', '9-2019-q4', '8-2018-q4']
```

Comment:
perhaps we can remove gcc 8 (and 9?) and add 13 instead?

Reply:
I left 9-2019-q4 because it is available in Ubuntu 20.04 which is stil quite common,  we can drop 8 I guess.

I'll add GCC 13 when it becomes available via github action we are using (https://github.com/carlosperate/arm-none-eabi-gcc-action)